demo/src/app/components/+datepicker/demos/date-initial-state/date-initial-state.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'demo-datepicker-date-initial-state',
+  templateUrl: './date-initial-state.html'
+})
+export class DemoDatepickerDateInitialStateComponent {
+  bsValue: Date = new Date();
```

Comment:
remove type annotation
if type annotation isn't provided, typescript will use the type of the first assigned value